cms/static/js/models/xblock_info.js

```diff
@@ -172,13 +172,17 @@ function(Backbone, _, str, ModuleUtils) {
             return !this.get('published') || this.get('has_changes');
         },
 
-        canBeDeleted: function(){
+        /**
+         * Return true if section content is required to show.
+         * currently, check section for an entrance exam.
+        */
+        showSectionContent: function(){
             //get the type of xblock
-            if(this.get('override_type') != null) {
-                var type = this.get('override_type');
+            var type = this.getOverrideType();
 
-                //hide/remove the delete trash icon if type is entrance exam.
-                if (_.has(type, 'is_entrance_exam') && type['is_entrance_exam']) {
+            if(type !=null) {
+                //hide the delete/drag icon if type is entrance exam.
```

Comment:
I disagree with hiding the drag icon on the entrance exam because it is still possible to drag other sections before it.